apps/src/StudioApp.js

```diff
@@ -1053,13 +1053,13 @@ StudioApp.prototype.toggleRunReset = function(button) {
     // Note: Checking alwaysHideRunButton is necessary because are some levels where we never
     // want to show the "run" button (e.g., maze levels that are "stepOnly").
     run.style.display =
-      showRun && !this.config.alwaysHideRunButton ? 'inline-block' : 'none';
+      showRun && !this.config.alwaysHideRunButton ? 'grid' : 'none';
```

Comment:
just a note for future us: i did a quick Windows 10 / IE11 test for this change (because IE11 doesn't really support CSS grid), and there's no visual difference:

<img width="1179" alt="Screen Shot 2022-01-05 at 10 54 45 AM" src="https://user-images.githubusercontent.com/9812299/148273050-d855173c-d562-45b4-8891-86fb66d67bab.png">

because we've deprecated support for IE11 and changing the display of this element from 'inline-block' to 'grid' has no visual regression, i think we should keep this as-is (rather than trying to find a solution that also fixes IE)